website/docs/components/accordion/index.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Accordion
-description: An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
-caption: A list of toggles that reveal or hide associated sections of content.
+description: (CV -) An accordion allow users to expand or collapse multiple sections of content in a stack. (KB -) An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
+caption: (CV -) Allow users to display multiple sections of content in a stack. (KB -) A list of toggles that reveal or hide associated sections of content.
 status: released
 links:
   figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/nGGr1ZOkVEPvzmq4HkASFn/HDS-Product---Components?type=design&node-id=36870-71031&t=JByoqnVP07zC5rEL-0
   github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/accordion
 previewImage: assets/illustrations/components/accordion.jpg
 navigation:
   hidden: false
-  keywords: ['reveal', 'toggle', 'disclosure', 'summary', 'details']
+  keywords: ['toggle', 'disclosure', 'details', 'reveal', 'summary', 'expand', 'collapse']
```

Comment:
@cveigt I merged together keywords I had added in the main documentation PR with ones you had. Please edit as you see fit (and as others suggest).